style.css

```
@@ -0,0 +1,494 @@
+*{
+    margin: 0 auto;
+    padding: 0;
+
+    box-sizing: border-box;
+}
+body{
+    position: relative;
+
+    width: 100vw;
+    height: 100vh;
```

All of the extra contents in your body is flowing out of the body. Use Inpect Element and see by hovering on the body, OR else you can give a border / outline to check the boundary of your box element. Hence we always use a relative value like min-height. So our height isn't fixed to 100vh but minimum 100vh, and it will keep on increasing as our contents keep getting larger / bigger.
```suggestion
    min-height: 100vh;
```

---

i notice that mistake of mine and its diffrence after your review... thanks for clearify the diffrance between relative and fix vlaues in responsive design.